### PR TITLE
Running gradle build no longer triggers a native build

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: docs/src/main/asciidoc/gradle-tooling.adoc [hunk 1]
```diff
@@ -329,8 +329,4 @@ The entries are relative to the root of the generated Uber-Jar. You can specify
 
 == Building with `./gradlew build`
 
-By default, `./gradlew build` will build the native image also. To skip it, use
-```
-./gradlew build -x buildNative -x testNative
-```
-
+Starting from 1.1.0.Final, `./gradlew build` will no longer build the native image. Use the `buildNative` task explicitly as explained above if needed.
```
Comment (geoand) [inline docs/src/main/asciidoc/gradle-tooling.adoc]: How about doing native tests? I think that needs to be documented as well

File: docs/src/main/asciidoc/gradle-tooling.adoc [hunk 2]
```diff
@@ -309,6 +309,18 @@ The native executable would then be produced by executing:
...
+== Running native tests
+
+Run the native tests using:
+
+[source,shell]
+----
+./gradlew testNative
+----
+
+This task depends on `buildNative`, so it will generate the build image before running the tests.
```
Comment (geoand) [inline docs/src/main/asciidoc/gradle-tooling.adoc]: :+1:

Comment (gastaldi) [reply docs/src/main/asciidoc/gradle-tooling.adoc]: s/build image/native image

Comment (geoand) [reply docs/src/main/asciidoc/gradle-tooling.adoc]: certainly :)

**Review by gsmet — APPROVED** [review]

**Pull request merged**